bears/general/KeywordBear.py

```diff
@@ -19,7 +20,11 @@ def _get_comments(dependency_results):
         return
 
     for result in annotation_bear_results:
-        yield from result.contents.get('comments', [])
+        if isinstance(result.contents, str):
+            logging.error(result.contents)
+            yield []
```

Comment:
As it was yeilding an empty list when `comments` key was not found in request.contents I thought I had to do the same in case when this execption arises.
Fine I'll remove it :+1: